eng/pipelines/ci.yml

```diff
@@ -4,6 +4,11 @@ trigger:
 pr: none
 
 jobs:
+#
+# Checkout repository
+#
+- template: /eng/checkout-job.yml
```

Comment:
It's a little bit unfortunate that we have to add this step into every pipeline yaml file. But I can't think about a better solution.